activitypub.go

```diff
@@ -77,18 +77,11 @@ func handleFetchCollectionActivities(app *App, w http.ResponseWriter, r *http.Re
 	vars := mux.Vars(r)
 	alias := vars["alias"]
 
-	// TODO: enforce visibility
-	// Get base Collection data
```

Comment:
We should retain this TODO comment, as this PR doesn't solve it.

Also, it probably wouldn't hurt to keep this informative comment below the TODO, either (unless you have a particular reason for removing it).

Reply:
sorry, this was accidental. Will add it back. 

Is there anything I can add to the "enforce visibility" to make it more descriptive? It is unclear to me what it implies. Maybe I can link an issue?